### ISSUE TITLE
Documentation: from dbt-ga4 to a dashboard

### ISSUE THREAD

**Issue by novica** [Velir/dbt-ga4#146]
I understand this is not an issue directly related to the package, and please feel free to close it right away if you think this is not a place to ask such a question. 

Are there plans to provide documentation how to use the models created from the package to make a simple dashboard (maybe just the few screens that are available in GA)? 

As far as I can see, there are no resources out there on how to use or what to do with the data once they are transformed in bigquery.

**Comment by adamribaudo-velir** [comment]
@novica I agree with you that there are too few resources on using GA4 & GA3 data in a BI and data warehouse environment. Most resources assume that you're going to use the native Looker Studio connectors. There are many issues that would be worth covering:
- How to get accurate, deduped user counts
- How to filter across tables
- How to handle reporting on segments/audiences
- Which metrics might differ from native GA4 and why

I believe this is all outside of the scope of the package, but it's interesting food for thought. I'm presenting the package at a dbt meetup on 3/22 that will be live-streamed: https://www.linkedin.com/video/event/urn:li:ugcPost:7034261432913252352/

I plan to show a sample dashboard as part of that talk. Hope that helps!

**Comment by novica** [comment]
@adamribaudo-velir Thanks for the follow-up. Do you know if there will e a recording of the event? I am in Europe and I seems in my timezone the event is going to be around midnight.

**Comment by afl5082** [comment]
@adamribaudo-velir hey Adam, do you have-all have the recording for that event? And or would you mind sharing the dashboard?

**Comment by adamribaudo-velir** [comment]
Funny story... the live stream was 'Zoom Bombed' with someone streaming pornographic content so they had to end it half-way through my talk and I'm guessing it won't be released. 

The talk went great otherwise :)

And I'm sorry but my sample dashboard displays real data from velir.com so I don't believe I can share that publicly. 

What I can do is share the aggregate models I used for the dashboard which I've posted here: 
https://github.com/Velir/dbt-ga4/blob/agg-models/models/marts/core/agg_pageviews_daily.sql
https://github.com/Velir/dbt-ga4/blob/agg-models/models/marts/core/agg_sessions_daily.sql

Everything in the dashboard was made from those 2 models. I'll also upload a PDF of the talk: [Adam Ribaudo - dbt-ga4 presentation.pdf](https://github.com/Velir/dbt-ga4/files/11073352/Adam.Ribaudo.-.dbt-ga4.presentation.pdf)

Hope that helps!

**Comment by afl5082** [comment]
@adamribaudo-velir 
haha oh no! That stinks. 

No worries, great thank you for pointing me to the models and sharing the pdf. This stuff is fantastic!!
